source/inputCore.py

```diff
@@ -677,7 +677,7 @@ def addObj(self, obj, isAncestor=False):
 				scripts[script] = scriptInfo
 		for gesture, script in obj._gestureMap.items():
 			try:
-				scriptInfo = scripts[script.__func__]
+				scriptInfo = scripts[script]

```

Comment:
Just documenting on the pr here that the reason is that script in Python2 was an unbound method. In Python3, unbound methods are normal functions.